### DISCUSSION TITLE
请问如何限制首页文章的长度呢？

### DISCUSSION THREAD

**Discussion by CJYKK** [ppoffice/hexo-theme-icarus#1162] [Q&A] [answered]
经常写一些多图的文章，这时候在首页的长度就太长了，别的文章就靠下了……

类似这样：
![screencapture-localhost-4000-2022-12-26-13_26_45](https://user-images.githubusercontent.com/59359590/209506524-9fdadf2c-15e2-431e-8c36-99a6c2ddfb27.png)

有没有什么方法能让首页的文章预览超过一定长度时就自动折叠一些呢？

**Comment by flipwind** [comment, marked as answer]
## 你可以使用 ``<!-- MORE -->`` 来设置简介 (｡･∀･)ﾉﾞ
```markdown
---
title: 你好
---
本文介绍了一个介绍
<!-- MORE -->
介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍
介绍介绍介绍介绍介绍介绍介绍介绍
介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍
介绍介绍介绍介绍介绍介绍介绍介绍
```
> 然后，主页显示的就是这样子：（示例）
---
> POSTED | 2秒前发布 | 1分钟读完
> # 你好
> ##### 本文介绍了一个介绍
---

**Reply by flipwind** [reply]
@CJYKK

**Reply by CJYKK** [reply]
我去试试

**Reply by CJYKK** [reply]
> ## 你可以使用 `<!-- MORE -->` 来设置简介 (｡･∀･)ﾉﾞ
> ```gfm
> ---
> title: 你好
> ---
> 本文介绍了一个介绍
> 
> 介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍
> 介绍介绍介绍介绍介绍介绍介绍介绍
> 介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍介绍
> 介绍介绍介绍介绍介绍介绍介绍介绍
> ```
> 
> > 然后，主页显示的就是这样子：（示例）
> 
> > POSTED | 2秒前发布 | 1分钟读完
> > # 你好
> > ##### 本文介绍了一个介绍

成功了，谢谢(｡･∀･)ﾉﾞ

**Comment by am-abudu** [comment]
作为楼上的补充，你还可以添加 excerpt 字段来作为首页的显示内容
